sympy/combinatorics/tests/test_coset_table.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
 from sympy.combinatorics.fp_groups import FpGroup
 from sympy.combinatorics.coset_table import (CosetTable,
-                    coset_enumeration_r, coset_enumeration_c)
+                    coset_enumeration_r, coset_enumeration_c, modified_scan_and_fill,
+                    modified_coincidence, modified_define, modified_merge, modified_rep, modified_scan)
```

Comment:
The last line seems very long, and I'm not sure if the one before it is the right length. Are they <=72 symbols per line (including whitespace at the start)?

Reply:
This is removed anyway, 'coz the modified methods need not be imported now.